src/components/HomepageFeatures/index.tsx

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
-import clsx from 'clsx';
-import styles from './styles.module.css';
+import React from "react";
```

Comment:
@nguyenoanh5597 
Hi,

I think we should keep the coding convention is single-quote

Reply:
fixed